### ISSUE TITLE
Improve display of two language names in post composer

### ISSUE THREAD

**Issue by surfdude29** [bluesky-social/social-app#3638]
**Is your feature request related to a problem? Please describe.**

If a user selects two languages for their post, the name of one of the languages will be cut off and it just doesn't look good. Given that feeds are often filtered by language, it is important to make it as easy, convenient and attractive as possible for users to tag their posts with the correct language(s).

This is an example of how the composer currently looks if a user has selected English and Japanese as the post languages:

![IMG_4027](https://github.com/bluesky-social/social-app/assets/149612116/cfd0df17-21a5-410a-84a0-a916b65976f4)

**Describe the solution you'd like**

When two languages are selected for a post, I would like the names of both languages to be displayed in full (space permitting), with either `and` or a `+` sign connecting them.

I made a mockup of how it might look using the `+` sign:

![IMG_4030](https://github.com/bluesky-social/social-app/assets/149612116/6ae5ba41-4082-4d96-b11b-3cca0f35b616)

**Describe alternatives you've considered**

An alternative is to simply keep the current behaviour, with the undesirable look previously described.

Another alternative is to display both post languages stacked on top of one another, although this would look worse I think.

(I realise that more than two languages can be selected for a post, but I'm assuming that that's rarely used).

**Comment by omar0404** [comment]
what about showing languages codes  e.x "en,es,ar" ?

**Comment by surfdude29** [comment]
> what about showing languages codes e.x "en,es,ar" ? @surfdude29

Yeah that could work too I guess, although I feel it's probably a bit less user-friendly, as I'm guessing that fewer users know what the language codes are for the languages they usually write in vs. the language's name in English.
